spec/generator.json

```diff
@@ -31,10 +31,19 @@
             ]
         },
 
+        "source_file" {
+            "description": "Represents a source file",
+            "fields": [
+                { "name": "filename", "type": "string", "description": "The recommended name for the file, including the path to the file, where appropriate.", "example": "com/myorg/myapi/Foo.java" },
+                { "name": "source", "type": "string", "description": "The actual source code." }
```

Comment:
I also think it might be worth adding a required content_type field now - maybe we make it an enum and just list "txt" for now - would leave open possibility in future of a zip file... wdyt?